src/hooks/useRenderCampaigns.tsx

```diff
@@ -19,14 +19,16 @@ function RecommendationComponentWrapper(props: {
   })
 }
 
+function injectPlacements(data: Record<string, unknown>) {
+  nostojs(api => api.placements.injectCampaigns(data as Parameters<API['placements']['injectCampaigns']>[0]))
```

Comment:
The type assertation looks a bit hairy but I'm unsure whether we have better options here

Reply:
you are right, but the fixes would be in this order
* fix in client script (alignment between Session API Action Response & placement types)
* update to nosto-js
* update to nosto-react